components/centraldashboard/public/components/resources/kubeflow-icons.js

```diff
@@ -6,7 +6,20 @@ import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
 
 import {html, htmlLiteral} from '@polymer/polymer/lib/utils/html-tag.js';
-import bolt from '../../assets/icons/ic_bolt.svg';
+
+/**
+ * Converts an imported folder into a Object.entries() type of
+ *  [[key, val], ...] list.
+ * @param {object} context
+ * @return {[...[string, string]]}
+ */
+function createFileMap(context) {
+    const keys = context.keys();
+    const values = keys.map(context);
+    const _v = (v) => stripSVG(v.default || v);
+    const _name = (n) => n.split(/\/|\\/).slice(-1)[0].replace('.svg', '');
+    return keys.map((k, i) => [_name(k), _v(values[i])]);
+}
 
 /**
  * Takes SVG source returns its contents.
```

Comment:
Can you clarify the intent of the regexes in the comment block? It seems like it they strip any excess whitespace and then trim anything but the content inside the SVG tag, but it's nice to not have to read them haha.

Reply:
Just did! Sorry about that :)